Sources/Initial/InitialCore/InitialCore.swift

```diff
@@ -27,8 +27,17 @@ public struct Initial: ReducerProtocol {
                 return .task {
                     .startRespond(
                         await TaskResult {
-                            try await self.rawgClient.getList(
-                                .init(page: 1, pageSize: 30, search: "")
+                            
+                            let string = "01/02/2020"
+                            let dateFormatter = DateFormatter()
+                            dateFormatter.dateFormat = "dd/MM/yy"
+                            
+                            return try await self.rawgClient.getList(
+                                .init(
+                                    page: 1,
+                                    pageSize: 30,
+                                    dates: (startDate: dateFormatter.date(from: string)!, endDate: Date())
```

Comment:
non riusciamo a togliere il bang in nessun modo? 😄

Reply:
yep fixato